mmrazor/models/distillers/configurable_distiller.py

```diff
@@ -237,7 +237,7 @@ def _check_loss_forward_mappings(
             loss_module = losses[loss_name]
             loss_forward_keys = signature(
                 loss_module.forward).parameters.keys()
-            assert len(loss_forward_keys) == len(forward_mappings.keys())
+            # assert len(loss_forward_keys) == len(forward_mappings.keys())
```

Comment:
parameters without default value have to be checked.

Reply:
fixed.